io-functions-pay-portal/.devops/code-review-be.yml

```diff
@@ -0,0 +1,89 @@
+# Azure DevOps pipeline to build, check source codes and run tests.
+#
+# To make Danger JS run on a pull request you need to add the following pipeline 
+# variable and set it with a GitHub access token (scope public_repo); otherwise 
+# set its value to 'skip' without marking it secret: 
+# - DANGER_GITHUB_API_TOKEN
+# 
+
+variables:
+  NODE_VERSION: '12.19.1'
+  YARN_CACHE_FOLDER: $(Pipeline.Workspace)/.yarn
+
+# Automatically triggered on PR
+# https://docs.microsoft.com/en-us/azure/devops/pipelines/yaml-schema?view=azure-devops&tabs=schema%2Cparameter-schema#pr-trigger
+trigger:
+  branches:
+    include:
+    - master
+    - main
+    - 175899260-integration-test
+  paths:
+    include:
+    - io-functions-pay-portal
+
+# Execute agents (jobs) on latest Ubuntu version.
+# To change OS for a specific, ovverride "pool" attribute inside the job definition
+pool:
+  vmImage: 'ubuntu-latest'
+
+stages:
+  - stage: Build
+    dependsOn: []
+    jobs:
+      - job: make_build
+        pool:
+          vmImage: 'ubuntu-latest'         
+        steps:
+        - template: azure-templates/setup-project.yml
+        - script: yarn build
+          workingDirectory: io-functions-pay-portal
+          displayName: 'Build'
+  
+  - stage: Static_analysis
+    dependsOn: []
+    jobs:
+
+      - job: lint     
+        steps:
+        - template: azure-templates/setup-project.yml
+        - script: yarn lint
+          workingDirectory: io-functions-pay-portal
+          displayName: 'Lint'
+      - job: validate_api_specification     
+        steps:
+        - script: npx oval validate -p io-functions-pay-portal/openapi/index.yaml
+          displayName: 'Validate API specification'
+      # - job: danger
```

Comment:
why is this commented out?

Reply:
I don't have the danger token, so I can't set up the danger control. I left it commented as a reminder

Reply:
I've set the secret token on the pipeline configuration. 
Now you need to add the Danger job to the pipeline. You can just copypaste from https://github.com/pagopa/io-functions-template/blob/master/.devops/code-review-pipelines.yml#L53

Reply:
@balanza Please note that using the template you suggested - at least in that repo - results in the following weird error during the danger job:

`TypeError: global$1.XMLHttpRequest is not a constructor`

It seems to be related to a wrong `gitlab` version - an indirect dependency - which is fixed by upgrading danger to `danger@10.xx `. Please see the following reference: 

https://github.com/danger/danger-js/issues/884